packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -1616,4 +1616,66 @@ describe('ReactSuspenseWithNoopRenderer', () => {
 
     expect(ReactNoop.getChildren()).toEqual([span('Loading A...')]);
   });
+
```

Comment:
Can we add a variant of this test that tests this with an update explicitly scheduled at Idle priority?